include/blmc_robots/blmc_joint_module.hpp

```diff
@@ -106,18 +106,25 @@ class BlmcJointModule
     /**
      * @brief Set the joint torque to be sent.
      * 
-     * @param desired_torque 
+     * @param desired_torque (N/m)
      */
     void set_torque(const double& desired_torque);
 
     /**
      * @brief Set the zero_angle. The zero_angle is the angle between the
      * closest positive motor index and the zero configuration.
      * 
-     * @param zero_angle 
+     * @param zero_angle (rad)
      */
     void set_zero_angle(const double& zero_angle);
 
+    /**
+     * @brief Define if the motor should turn clock-wize or counter clock-wize.
+     * 
+     * @param should_reverse true:reverse rotation axis, false:do nothing.
+     */
+    void set_joint_polarity(const bool& should_reverse);
```

Comment:
should_reverse seems not very descriptive to me, could we call it reverse_polarity?